new_lineage_panel/src/graph.ts

```diff
@@ -635,3 +716,41 @@ export const bfsTraversal = async (
   }
   return isLineage;
 };
+
+export const moveTableFromSeeMoreToCanvas = (
```

Comment:
some comments for each of these methods will be useful. I dont think anyone except @AdiGajbhiye can fix issues in this flow :)

Reply:
Will be really useful if we have unit tests atleast, so we could have documented different usecases handled in these methods @anandgupta42